src/category_theory/triangulated/rotate.lean

```diff
@@ -203,27 +238,40 @@ def rot_comp_inv_rot : 𝟭 (triangle C) ≅ rotate ⋙ inv_rotate :=
 { hom := rot_comp_inv_rot_hom,
   inv := rot_comp_inv_rot_inv }
 
+/-- There is a natural map from the `rotate` of the `inv_rotate` of a triangle to itself. -/
+@[simps]
+def inv_rot_comp_rot_hom_app (T : triangle C) : rotate.obj (inv_rotate.obj T) ⟶ T :=
```

Comment:
It seems like the names you chose for these lemmas will clash with those generated by `simps` later on, which is not ideal as we will get weird names for lemmas like `foo_1`.